cluster.go

```diff
@@ -135,6 +150,7 @@ func NewCluster(clusterAddrs []string, opts ...ClusterOpt) (*Cluster, error) {
 	defaultClusterOpts := []ClusterOpt{
 		ClusterPoolFunc(DefaultClientFunc),
 		ClusterSyncEvery(5 * time.Second),
+		ClusterWaitWhenDown(100 * time.Millisecond),
```

Comment:
Random magic number incoming. This is just a random value that I came up with when prototyping, which worked great with the artificial delays I configured for my local cluster.